catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -31,6 +31,8 @@
             required="false" type="String"/>
         <AD description="Password for WFS Service (optional)" name="Password" id="password"
             required="false" type="Password"/>
+        <AD description="Force only a specific FeatureType to be queried instead of all featureTypes"
+            name="Force Feature Type" id="forceFeatureType" type="String" required="false"/>
```

Comment:
UX Nit: Suggest renaming "Force" here to "Forced" to be consistent with the "Forced Spatial Filter Type" visible in the user interface screen? Also, recommend putting the Forced Feature and Forced Spatial Filter types back to back since they are similar options.